draw_party/handlers/test/test_handlers.py

```diff
@@ -0,0 +1,90 @@
+import falcon
+
+from draw_party import handlers
+from draw_party.test import BaseTestCase, Mock, call, patch
+
+
+class JsonEndpointTestCase(BaseTestCase):
+    def setUp(self):
+        super(JsonEndpointTestCase, self).setUp()
+        self.mock_log = patch.object(
+            handlers, 'log', autospec=True).start()
+        self.mock_response = Mock(spec=falcon.Response)
+
+        self.req_endpoint = handlers.JsonEndpoint(required_params={
+            'group_id': int,
+            'username': str
+        })
+        self.no_req_endpoint = handlers.JsonEndpoint()
+
+    def test_json_post_req_params(self):
+        resp = self.req_endpoint.load_params(
+            Mock(falcon.Request,
+                 content_length=24,
+                 stream=Mock(
+                     read=lambda: '{"group_id": 12, "username": "test"}')),
```

Comment:
is this equivalent to 
`read=Mock(return_value='{"group_id": 12, "username": "test"}'` ?

Reply:
Yes, just shorter and less mock in mock creation

Reply:
I like it